app/com/linkedin/drelephant/util/SparkUtils.scala

```diff
@@ -274,6 +282,9 @@ trait SparkUtils {
       // if codec is not available, but we found a file match with appId, use the actual file Path from the first match
       case nocodec if nocodec._1 != None & nocodec._3 == None => (attemptsList(0).getPath(), DEFAULT_COMPRESSION_CODEC)
 
+      case nocompression if nocompression._1 == None & nocompression._2 == None & nocompression._3 == None
```

Comment:
Please write the coment for this case condition , as have written in another case conditions